packages/nx-monorepo/README.md

```diff
@@ -94,5 +94,72 @@ The NxMonorepoProject also manages your yarn/pnpm workspaces for you and synthes
 
 For more information on NX commands, refer to this [documentation](https://nx.dev/using-nx/nx-cli).
 
+### Homogenous Dependencies
 
+As well as adding implicit dependencies, you can add dependencies between projects of the same language in order to have one project consume another project's code.
 
+#### Typescript
+
+Since the `NxMonorepoProject` manages a yarn/npm/pnpm workspace, configuring dependencies between Typescript projects is as straightforward as referencing them in deps:
```

Comment:
We should also mention that if trying to add deps between projects before projects are first synthed, you will run into an error as the project wont be present in the workspace yet and it will attempt to pull it down from NPM where it won't exist.

Reply:
Good call!